packages/multichain-network-controller/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    { "path": "../base-controller" },
+    { "path": "../accounts-controller" },
+    { "path": "../network-controller" }
+  ],
+  "include": ["../../types", "./src"],
+  /**
+   * Here we ensure that TypeScript resolves `@metamask/*` imports to the
+   * uncompiled source code for packages that live in this repo.
+   *
+   * NOTE: This must be synchronized with the `moduleNameMapper` option in
+   * `jest.config.packages.js`.
+   *
+   * NOTE 2: This is not necessary when copying this package to `packages/`.
+   */
+  "paths": {
+    "@metamask/*": ["../../packages/*/src", "../*/src"]
+  }
```

Comment:
```suggestion
 
```

Reply:
Agreed, we should not need these changes as they are provided by `tsconfig.packages.json`.